packages/@lwc/synthetic-shadow/src/polyfills/window-event-target/polyfill.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { windowRemoveEventListener, windowAddEventListener } from '../../env/window';
```

Comment:
```suggestion
import { defineProperties, isUndefined, isTrue, ArraySlice } from '@lwc/shared';
import { windowRemoveEventListener, windowAddEventListener } from '../../env/window';
```

Reply:
I'm ok with this, first time I heard about this though.